api/tasks.py

```diff
@@ -11,3 +14,22 @@ def create_delivery(order):
     else:
         delivery.print_pdf()
         return {'success': 'all good'}
+
+
+@app.task
+def update_customer_if_is_partner(email):
+    if _check_customer_is_partner(email):
+        # here will come the code to update the Wordpress user role.
+        pass
+
+
+def _check_customer_is_partner(email):
+    customer = Customer().fetch(email)
+    # if the user in the crm has a role that it is considered as a LaZona partner/membership
```

Comment:
The fact that you need to point this out in a comment is a clear symptom IMO that this needs to become a self-explanatory method, which will make the comment redundant and unnecessary. It doesn't need to be a new class just yet. Just a private method down here will do it for now. As the app evolves, we'll see where this and other methods need to be moved to.